pythia/modules/metrics.py

```diff
@@ -312,6 +313,79 @@ def calculate(self, sample_list, model_output, *args, **kwargs):
         return accuracy
 
 
+@registry.register_metric("vqa_evalai_accuracy")
+class VQAEvalAIAccuracy(BaseMetric):
```

Comment:
I guess easiest way to use this would be to override command line args for metrics and then call specifically only val evaluation. We should ponder over better method to do this.

Reply:
Do you mean adding the ability to specify a particular metric as a flag argument or just use the `vqa_evalai_accuracy` metric by default for validation on vqa?